frontend/src/reports/Report.tsx

```diff
@@ -86,6 +90,17 @@ export function Report(props: ReportProps) {
     setCurrentBreakdown(str)
   }
 
+  const demographicOptionsMap = getDemographicOptionsMap(dataTypeConfig)
+
+  // if (!Object.values(demographicOptionsMap).includes(currentBreakdown)) {
+  //   console.log(JSON.stringify(demographicOptionsMap), "doesn't include", currentBreakdown);
+
+  //   setDemoWithParam(Object.values(demographicOptionsMap)[0] as BreakdownVar)
+  // }
```

Comment:
hmmm i dunno. i'll uncomment and see what happens lol, that's problem with these long running PRS.... i dont even know what this is doing because i wrote this months ago